homeassistant/components/cover/mqtt.py

```diff
@@ -190,3 +254,35 @@ def async_stop_cover(self, **kwargs):
         mqtt.async_publish(
             self.hass, self._command_topic, self._payload_stop, self._qos,
             self._retain)
+
+    @asyncio.coroutine
+    def async_open_cover_tilt(self, **kwargs):
+        """Tilt the cover open."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_open_position, self._qos, self._retain)
+
+    @asyncio.coroutine
+    def async_close_cover_tilt(self, **kwargs):
+        """Tilt the cover closed."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_closed_position, self._qos, self._retain)
+
+    @property
+    def current_cover_tilt_position(self):
```

Comment:
Move this property up to be next to its other property friends. It's lonely down here.